scripts/ci

```diff
@@ -13,4 +13,4 @@ fi
   ./script/ci
 )
 
-sbt clean compile test riffRaffUpload
+sbt clean scalafmtCheckAll compile test riffRaffUpload
```

Comment:
Less important but can also include `scalafmtSbtCheck` for SBT config.
https://scalameta.org/scalafmt/docs/installation.html#task-keys

Reply:
Ah great shout, hadn't noticed that wasn't covered. Will do.